src/sqlfluff/dialects/dialect_clickhouse.py

```diff
@@ -511,10 +511,111 @@ class DatatypeSegment(BaseSegment):
 
     type = "data_type"
     match_grammar = OneOf(
+        # Nullable(Type)
         Sequence(
-            StringParser("NULLABLE", CodeSegment, type="data_type_identifier"),
+            StringParser("Nullable", CodeSegment, type="data_type_identifier"),
```

Comment:
Our pattern generally for StringParser() is to uppercase the contents. Can you change your ones to match this?